README.md

```diff
@@ -198,8 +198,10 @@ JSON support is fairly minimal in that you can call a `pymarc.Record`'s
 ```python
 from pymarc import MARCReader
 
-record = MARCReader('test/one.dat')
-print(record.as_json(indent=2))
+with open('test/one.dat','rb') as fh:
```

Comment:
The fact that the original code in the README for years couldn't possibly have worked may be an argument in favor of adding doctests :)